### PR TITLE
add Observability and Analysis category

### PR DESCRIPTION
**Pull request by MarckK**
### Pre-submission checklist:

*Please check each of these after submitting your pull request:*

* [ ] Are you only including a `repo_url` if your project is 100% open source? If so, pick the single best GitHub repository for your project, not a GitHub organization.
* [ ] Does it follow the guidelines for [new entries](https://github.com/cdfoundation/cdf-landscape#new-entries)?
* [ ] Have you added an SVG to `hosted_logos` and referenced it there?
* [ ] Does your logo clearly state the name of the project/product and follow logo [guidelines](https://github.com/cdfoundation/cdf-landscape#logos)?
* [ ] Have you verified that the Crunchbase data for your organization is correct (including headquarters and LinkedIn)?
* [ ] ~5 minutes after opening the pull request, the CNCF-Bot will post a link for you to preview your changes. Have you confirmed that they are correct and then added a comment to the PR saying "LGTM"?

### CODE REVIEW

**Review by markyjackson-taulia — APPROVED** [review]

**Comment by MarckK** [comment]
This is an entirely trial PR, to map out landscape category changes discussed at landscape wg meeting and https://github.com/cdfoundation/cdf-landscape/issues/195.

There remains too much overlap on the landscape with the categories proposed in PR, especially around 
monitoring:

Screenshot 2021-11-12 at 23 47 31


Additional Note: So as to avoid having to restructure the layout of landscape, I replaced the current `Change Management` category with `Observability and Analysis`. This is a temporary hack to be able to easily play around with new category.  There is no intention to remove `Change Management` category from the CDF landscape at this time.

**Comment by MarckK** [comment]
Hi @tdcox @sbtaylor15 @TracyRagan @tracymiranda @oleg-nenashev,
I'm looking for further feedback and input in this section  (see comments and screenshot above).

Open to suggestions to refine naming and categorisation in this area!

**Comment by tdcox** [comment]
Architecturally, it would be normal to treat Security and Observability as different categories, but include Monitoring within Observability.

**Comment by MarckK** [comment]
Thank you, @tdcox! 
Sometimes I do need to be given implied permission, or encouragement, to go change someone else's prior work 😅 

Currently, the prototype (as can be viewed in preview) looks like:


Screenshot 2021-11-19 at 22 46 58

We'll not discuss DevSecOps/Security in this PR, although that does need to be addressed separately.

For changes to the categories shown in the above screen shot, I'd likely put Monitoring & Logging in one category. 
What other changes would you suggest?

**Comment by tdcox** [comment]
That looks better. At this point we start to get into nuanced conversations about monitoring being an active process and logging being a passive one…

Pragmatically, you will need to set a granularity for the document and make the decision to lump stuff together at that granularity. In reality, it’s fractal all the way down and you can continue to sub-categorise everything, the closer you get to it.

**Comment by MarckK** [comment]
Building in the feedback we've had so far, the section now looks like:
Screenshot 2021-11-26 at 16 23 17

This is still a prototype for the section and **any and all feedback is welcome!**

**Comment by e-backmark-ericsson** [comment]
CD Events should go into the Tracing and Messaging sub category in accordance with this proposal, once the needed data for CD Events is set up (#192)

**Review by oleg-nenashev — APPROVED** [review]
The CDF Events SIG meeting supports this proposal (Dec 06, 2021)

File: landscape.yml [hunk 1]
```diff
@@ -462,62 +449,63 @@ landscape:
...
-            organization:
-              name: Gitea
-          - item:
-            name: GitHub
-            homepage_url: https://github.com/
-            logo: git-hub.svg
-            crunchbase: https://www.crunchbase.com/organization/github
+        - item:
+          name: Four Keys
+          homepage_url: https://cloud.google.com/blog/products/devops-sre/the-2019-accelerate-state-of-devops-elite-performance-productivity-and-scaling
```
Comment (dinagraves) [inline landscape.yml:460]: I would use one of the following: 

- https://cloud.google.com/blog/products/devops-sre/using-the-four-keys-to-measure-your-devops-performance
- https://cloud.google.com/devops/state-of-devops
- https://cloud.google.com/blog/products/devops-sre/announcing-dora-2021-accelerate-state-of-devops-report

File: landscape.yml [hunk 2]
```diff
@@ -462,62 +449,63 @@ landscape:
...
+          repo_url: https://github.com/GoogleCloudPlatform/fourkeys
+          logo: key.svg
```
Comment (davidstanke) [inline landscape.yml:462]: I've opened a PR to add the logo -- note that the filename is `fourkeys.png`

**Pull request merged**